### PR TITLE
Produce oom issue

### PR DESCRIPTION
**Pull request by nekhtiari**

### CODE REVIEW

File: blocks/s2_superresolution/Dockerfile [hunk 1]
```diff
@@ -2,6 +2,17 @@
 
 # Use one of the official Tensorflow Docker images as base.
 FROM tensorflow/tensorflow:latest-gpu-py3
+LABEL maintainer="NVIDIA CORPORATION <cudatools@nvidia.com>"
```
Comment (rodrigoalmeida94) [inline blocks/s2_superresolution/Dockerfile]: Is this addition required?

File: blocks/s2_superresolution/src/predict/s2_tiles_supres.py [hunk 2]
```diff
@@ -292,19 +288,20 @@ def data_final(
...
-                        row_off=y_mi,
-                        width=x_ma - x_mi + n_res,
-                        height=y_ma - y_mi + n_res,
-                    )
-                ),
-                0,
-                3,
-            )[:, :, term]
+            LOGGER.info(term)
+            with rasterio.open(data) as d_s:
```
Comment (rodrigoalmeida94) [inline blocks/s2_superresolution/src/predict/s2_tiles_supres.py]: Nice!

File: blocks/s2_superresolution/Dockerfile (hunk 1 above)
Comment (nekhtiari) [reply blocks/s2_superresolution/Dockerfile]: Actually no, I was experimenting with this when I (and pooja) was trying to solve memory issues to update the cuda library within the docker container that I was working in GC. I can remove it. It is not necessary for the block.
